README.md

```diff
@@ -1 +1,72 @@
-# Awesome-books
\ No newline at end of file
+# Awesome Books
+
+## Screenshot
+
+![screenshot](/images/app_screenshot.PNG)
+
+# Additional description about the project and its features.
+
+This is a project on javascript objects. The goal of this project is to test how to add and remove objects from local storage and get used to using Javascript objects.
+
+## Built With
+
+- HTML, CSS, Javascript
+- .Stylelintrc
+- .Hintrc
+- .eslintrc.json
+
+## Live Demo
+
+[Live Demo Link](http://darrenodi.me/Awesome-books/)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+1. First, Fork the repository then Clone it to your machine using the git clone command
+2. Install dependencies such as npm
+3. Make sure you have a code editor such as vscode or sublime text
+4. Once you're done cloning, you can use the ls command to view repo.
+5. You can now run this project locally
+
+### Prerequisites
+
+1. You would need a code editor such as vs code for this project
+2. A command line interface such as git bash is recommended
+
+## Authors
+
+👤 **Author 1**
```

Comment:
- [OPTIONAL] Kindly refactor Author 1, 2, and 3 to your real names. This authors' part would look professional that way.